lighthouse-core/report/html/report-styles.css

```diff
@@ -1339,6 +1345,10 @@
   padding-right: 0;
 }
 
+.lh-table tr {
+  vertical-align: baseline;
```

Comment:
it seems like this could get us in trouble applying to all table `tr`s--maybe there's a specific case we've been depending on the current behavior?--but I couldn't find any bad tables in the sample_v2 report so SGTM :)

Reply:
I did the same and couldn't find a place where a table is affected negatively. If it becomes an issues, I can always come back and make the formatting more specific.